app.js

```diff
@@ -9,25 +9,51 @@
   var handleResponse = function(e) {
     var items = JSON.parse(e.target.responseText);
 
-    var ul = document.createElement('ul');
-    ul.className = 'list--bare row';
-    var li, dominant, title;
-
     items.forEach(function(item) {
-      div = document.createElement('div');
-      div.className = 'palette';
+      var dominant, contrastingDominant, subDominant, contrastingSubDominant, pop
+      palette = document.createElement('div');
+      palette.className = 'palette'
+
+      keyword = document.createElement('h3');
+      keyword.innerHTML = item.keyword
+      palette.appendChild(keyword);
+
+
+      dominant = document.createElement('div');
+      dominant.className = "dominant";
+      dominant.innerHTML = item.colors.dominant;
+      dominant.style.backgroundColor = dominant.innerHTML
```

Comment:
make sure you include semi-colons. It's good practice and will save you in the future (and also not piss coworkers off)